ui/src/app/components/resources.tsx

```diff
@@ -11,11 +11,11 @@ import {Cohort, ConceptSet} from 'generated/fetch';
 export const ResourceCardMenu: React.FunctionComponent<{
   disabled: boolean, resourceType: ResourceType, onRenameNotebook?: Function,
   onRenameCohort?: Function, onOpenJupyterLabNotebook?: any, onCloneResource?: Function,
-  onDeleteResource?: Function, onEditCohort?: Function, onReviewCohort?: Function,
-  onEditConceptSet?: Function
+  onCopyResource?: Function, onDeleteResource?: Function, onEditCohort?: Function,
+  onReviewCohort?: Function, onEditConceptSet?: Function
 }> = ({
         disabled, resourceType, onRenameNotebook = () => {}, onRenameCohort = () => {},
-        onOpenJupyterLabNotebook = () => {}, onCloneResource = () => {},
+        onOpenJupyterLabNotebook = () => {}, onCloneResource = () => {}, onCopyResource = () => {},
```

Comment:
I think you need to add `onCopyResource` to the notebook case below.

Reply:
I left it out here so it wouldn't have any user facing changes. The follow up PR has the rest of the front end code.